### ISSUE TITLE
XMEM server JCL PARMLIB is set to SAMPLIB

### ISSUE THREAD

**Issue by John-A-Davies** [zowe/zowe-install-packaging#1034]
**Describe the bug**
After the XMEM JCL is tailored, its PARMLIB dataset name is changed to the SAMPLIB DSN instead of the PARMLIB DSN.

**To Reproduce**
1. run install-test
```
09:34:17 //ZWESIS01 EXEC PGM=ZWESIS01,REGION=&RGN,
09:34:17 //         PARM='NAME=&NAME,MEM=&MEM'
09:34:17 //STEPLIB  DD   DSNAME=ZOWEAD3.LOADLIB,DISP=SHR
09:34:17 //PARMLIB  DD   DSNAME=ZOWEAD3.ZWE.SZWESAMP,DISP=SHR
09:34:17 //SYSPRINT DD   SYSOUT=*
09:34:17 [install-zowe.sh] VENDOR.PROCLIB(ZWESISTC) ==== end of contents
```

**Expected behavior**
```
09:34:17 //ZWESIS01 EXEC PGM=ZWESIS01,REGION=&RGN,
09:34:17 //         PARM='NAME=&NAME,MEM=&MEM'
09:34:17 //STEPLIB  DD   DSNAME=ZOWEAD3.LOADLIB,DISP=SHR
09:34:17 //PARMLIB  DD   DSNAME=ZOWEAD3.PARMLIB,DISP=SHR
09:34:17 //SYSPRINT DD   SYSOUT=*
09:34:17 [install-zowe.sh] VENDOR.PROCLIB(ZWESISTC) ==== end of contents
```

**Comment by John-A-Davies** [comment]
Test run https://wash.zowe.org:8443/job/zowe-install-test/job/staging/742/consoleFull verifies that
the changes in https://wash.zowe.org:8443/job/zowe-install-packaging/job/zis-samplib/ worked.
```
10:00:43 //ZWESIS01 EXEC PGM=ZWESIS01,REGION=&RGN,
10:00:43 //         PARM='NAME=&NAME,MEM=&MEM'
10:00:43 //STEPLIB  DD   DSNAME=ZOWEAD3.LOADLIB,DISP=SHR
10:00:43 //PARMLIB  DD   DSNAME=ZOWEAD3.PARMLIB,DISP=SHR
10:00:43 //SYSPRINT DD   SYSOUT=*
```

**Comment by John-A-Davies** [comment]
Verifed that install-test fix #742 produces the correct result; see https://wash.zowe.org:8443/job/zowe-install-test/job/staging/749/
